task2/js/app.js

```diff
@@ -0,0 +1,33 @@
+document.addEventListener('DOMContentLoaded', () => {
+    let imageName = 'zombie-dead';
+    const imagePath = '/icons';
+    const extension = 'gif';
+    const zombieParentSelector = document.querySelector('.zombies');
+    const zombieSrc = `${imagePath}/${imageName}.${extension}`;
+
+    for (let zombie of zombies) {
+        const zombiePic = document.createElement('img');
```

Comment:
if you define CSS classes for each zombie this command should create DIV element instead of IMG